plugin-core/src/main/java/appland/javaAgent/AppMapJavaAgentDownloadService.java

```diff
@@ -98,28 +98,12 @@ public boolean downloadJavaAgentSync(@NotNull ProgressIndicator indicator) throw
         return true;
     }
 
-    @Nullable
-    private static Release.Asset getLatestAsset(@NotNull ProgressIndicator indicator) {
-        final Release[] releases = {MavenRelease.INSTANCE, GitHubRelease.INSTANCE};
-        for (Release release : releases) {
-            try {
-                var latestAsset = release.getLatest(indicator)
-                        .stream()
-                        .filter(asset -> "application/java-archive".equals(asset.getContentType()))
-                        .findFirst()
-                        .orElse(null);
-
-                if (latestAsset != null) {
-                    LOG.info("Got latest asset URL: " + latestAsset.getDownloadUrl());
-                    return latestAsset;
-                }
-            } catch (IOException e) {
-                LOG.info(String.format("Failed to query release, %s: %s", release, e));
-            }
-        }
+    public @NotNull Path agentDirPath() {
```

Comment:
Could you make this `getAgentDirPath()`, please?

Reply:
Done! :+1: